fastify.js

```diff
@@ -26,6 +26,7 @@ const {
   kPluginNameChain
 } = require('./lib/symbols.js')
 
+require('./lib/warnings')
```

Comment:
why is this needed?

Reply:
For load the warnings in memory. They are still declared in that file :)

Reply:
Is this how our current implementation works? The side effect nature of this is not great.

Reply:
I don't think this `require` is needed at all.

Reply:
@mcollina I fear I am missing something then. How would you load the warnings then?

@jsumners yes, currently the module is a singleton, I didn’t apply any changes from the original Fastify code. I was thinking about exposing a builder function tho.

Reply:
I think you can remove this line and all tests will pass. The warnings will be loaded by the other files that needs `emitWarning`.

Reply:
Nope, they will not :)
`fastify-warning` at the moment is a singleton, and if we remove this require, then the `createWaring` api will never be called. As result, `emitWarning` will start failing because the deprecations codes we are using have not been registered.